web/src/app/workplace/services/task.service.ts

```diff
@@ -1,18 +1,26 @@
-import { Task } from '../models/task';
-import { Observable } from 'rxjs/Observable';
-import { HttpClient, HttpHeaders } from '@angular/common/http';
-import { Injectable } from '@angular/core';
-import { environment } from 'app/../environments/environment';
+import {Task} from '../models/task';
+import {Observable} from 'rxjs/Observable';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {environment} from 'app/../environments/environment';
 
 @Injectable()
 export class TaskService {
   url = environment.taskanaRestUrl + '/v1/tasks';
```

Comment:
Please use template string for all strings with params, It's very usefull feature from modern programming languages, https://basarat.gitbooks.io/typescript/docs/template-strings.html
[This comment apply to all strings].